src/features/projectsV2/ProjectDetails/components/OtherInterventionInfo.tsx

```
@@ -173,24 +173,25 @@ const OtherInterventionInfo = ({
+        />
+      ),
     plantLocationInfo &&
-    plantLocationInfo.sampleInterventions &&
-    plantLocationInfo.sampleInterventions.length > 0 && (
-      <SampleTrees
-        key="sampleTrees"
-        sampleInterventions={plantLocationInfo.sampleInterventions}
-        setSelectedSamplePlantLocation={setSelectedSamplePlantLocation}
-      />
-    ),
+      plantLocationInfo.sampleInterventions &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)